### ISSUE TITLE
Enable coloured tablature symbols

### ISSUE THREAD

**Issue by reinierdevalk** [music-encoding/tablature-ig#22]
As seen in, e.g., the Capirola Ms and some of the vihuela books.

Examples:
`capirola-no_1.png`
![capirola-no_1](https://user-images.githubusercontent.com/14223345/114921670-19e2ac00-9e2b-11eb-8756-11c3806adaaa.png)

`fuenllana-1554_3-no_31.png` 
![fuenllana-1554_3-no_31](https://user-images.githubusercontent.com/14223345/114922977-945ffb80-9e2c-11eb-980d-bd9e06c00f15.png)

**Comment by DILewis** [comment]
I think we're ok for notes, because they're already able to take `@color`. We just need to remember to do the same for rhythm flags. There are rules for Capirola, but implementing those seems like overkill, given that it's unique for that.

**Comment by reinierdevalk** [comment]
@DILewis Yes - then this issue can serve as that reminder. I'm just adding everything I can think of.
